RecoTracker/LST/python/lstProducer_cff.py

```diff
@@ -0,0 +1,9 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoTracker.LST.lstProducer_cfi import lstProducer
+
+from Configuration.ProcessModifiers.gpu_cff import gpu
+(~gpu).toModify(lstProducer.alpaka, backend = 'serial_sync')
+
+from RecoTracker.LST.lstModulesDevESProducer_cfi import lstModulesDevESProducer
+(~gpu).toModify(lstModulesDevESProducer.alpaka, backend = 'serial_sync')
```

Comment:
The global behavior can be controlled with [`process.options.accelerators`](https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideEDMParametersForModules#The_options_Parameter_Set). By default (NVIDIA) GPU is used if such exists, and otherwise it falls back to CPU. You can set it explicitly along
```py
# force running on CPU
process.options.accelerators = ['cpu']

# force running on NVIDIA GPU
process.options.accelerators = ['gpu-nvidia']
```
More ways to customize the backends of modules are also shown in https://github.com/cms-sw/cmssw/blob/master/HeterogeneousCore/AlpakaCore/README.md#examples-on-explicitly-setting-the-backend 

`--accelerators` argument of `cmsDriver.py` controls this parameter directly.